src/LinuxTracing/TracerImpl.h

```diff
@@ -162,6 +166,7 @@ class TracerImpl : public Tracer {
   absl::flat_hash_map<uint64_t, uint64_t> uprobes_uretprobes_ids_to_function_id_;
   absl::flat_hash_set<uint64_t> uprobes_ids_;
   absl::flat_hash_set<uint64_t> uprobes_with_args_ids_;
+  absl::flat_hash_set<uint64_t> uprobes_of_functions_to_record_stack_ids_;
```

Comment:
I think this should just be `uprobes_with_stack_ids_` (just like the local variable `is_uprobe_with_stack`).